container/lxd/lxd_test.go

```diff
@@ -201,3 +203,39 @@ func (t *LxdSuite) TestNetworkDevicesWithParentDevice(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 	c.Assert(result, jc.DeepEquals, expected)
 }
+
+func (t *LxdSuite) TestGetImageSourcesDefaultConfig(c *gc.C) {
+	mgr := t.makeManager(c, t.baseConfig())
+
+	sources, err := lxd.GetImageSources(mgr)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Check(sources, gc.DeepEquals, lxdclient.DefaultImageSources)
+}
+
+func (t *LxdSuite) TestGetImageSourcesDefaultNonStandardStreamDefaultConfig(c *gc.C) {
+	cfg := t.baseConfig()
+	cfg[config.ContainerImageStreamKey] = "nope"
+	mgr := t.makeManager(c, cfg)
+
+	sources, err := lxd.GetImageSources(mgr)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Check(sources, gc.DeepEquals, lxdclient.DefaultImageSources)
+}
```

Comment:
shouldn't we have a test that just setting the ImageStream to "daily" returns only the second DefaultImageSource ?

Reply:
Yes, will add.